Solutions/LargestPrimeFactor.cs

```diff
@@ -0,0 +1,36 @@
+namespace Solutions;
+
+public class LargestPrimeFactor : ProblemSolverBase
+{
+  public LargestPrimeFactor()
+  {
+    ProblemID = 3;
+  }
+  public override void Solve()
+  {
+    Answer = GetLargestPrimeFactor(600851475143).ToString();
+  }
+
+  long GetLargestPrimeFactor(long number)
+  {
```

Comment:
I don't like 

foo()
{
}

I'm more of a 

foo () {
} 

guy myself, doesn't matter which we use for a solo dev project but you won't be a solo dev forever, can you add a linter and pick a style guide for c# that is wide spread enough to have proper linter support? And add that into the pre commit hooks, also setup github actions so that the CI/CD rejects the PRs before I do when it comes to linting issues. While you are at it, we don't have tests but we do have a Main which the CI/CD check to see if it runs properly, kinda like tests and if it exits with an error it again should reject the PR.